src/chat/components/bubble/CustomBubble.tsx

```diff
@@ -18,6 +18,8 @@ interface CustomBubbleProps {
   customTextStyle?: StyleProp<ViewStyle>;
   unReadSentMessage?: string;
   unReadSeenMessage?: string;
+  enableSeenMessage: boolean;
+  viewUnReadCustom?: (hasUnread: boolean) => JSX.Element;
```

Comment:
I think the name of this prop should be `customUnReadView` or `customViewUnRead`

Reply:
ok I renamed it